src/common/modules/Logger.js

```diff
@@ -13,6 +13,36 @@ const MESSAGE_LEVEL_NAME = Object.freeze({
     [MESSAGE_LEVEL.SUCCESS]: "SUCCESS"
 });
 
+/**
+ * Checks if value is a object.
+ *
+ * @function
+ * @param  {*} value
+ * @returns {boolean}
+ */
+function isObject(value){
+    const type = typeof value;
+    return value != null && (type === "object" || type === "function");
+}
+
+/**
+ * Copy nested objects to ensure a properly output.
+ *
+ * @function
+ * @param  {...*} args
+ * @returns {Object}
+ */
+function verifyArgs(args) {
```

Comment:
Better rename it, as it does not really verify any arguments. It more or less just "freezes" the objects(?), or "prepares the objects for logging". Or so? Just come up with a good name. 😄